samples/bluetooth/central_bas/prj.conf

```diff
@@ -3,6 +3,11 @@
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
+CONFIG_ASSERT=y
+CONFIG_ASSERT_LEVEL=2
```

Comment:
Assert level is 2 by default, skip this entry?

Reply:
Agree, remove if this is the default.